hermes-schema/src/main/java/pl/allegro/tech/hermes/schema/confluent/SchemaRegistryRawSchemaClient.java

```diff
@@ -81,13 +81,15 @@ public SchemaRegistryRawSchemaClient(Client client, URI schemaRegistryUri, Objec
     private Optional<String> extractSchema(String subject, String version, Response response) {
         switch (response.getStatusInfo().getFamily()) {
             case SUCCESSFUL:
+                logger.info("Found schema for subject {} at version {}", subject,  version);
```

Comment:
This stays? This will produce quite a lot of logs.

Reply:
Not so much, if schemas are not updated then one line per topic every N hours
From my experience It's *very* useful for debugging what actually happened when user updated schema.